src/clanhr/analytics/errors.clj

```diff
@@ -28,12 +28,19 @@
   (println request)
   (exception ex request))
 
+(defn object->hash-map
+  [obj]
+  (cond-> obj
+    (not (map? obj)) {:info obj}))
+
 (defn error
   "Registers a specific error"
   ([e] (error e {} {}))
   ([e info] (error e info {}))
   ([e info user]
-   (exception (RuntimeException. e) info user)
+   (if (instance? Exception e)
+     (exception (RuntimeException. e) info user)
+     (exception (ex-info "info" (object->hash-map e)) info user))
```

Comment:
I'd change the str from `info` to `RuntimeException` or something like that. That's the message we'll get when we do `.getMessage`, could be more descriptive.